### PR TITLE
Add CSS for sticky table headers

### PR DESCRIPTION
**Pull request by lukebjerring**
Fixes https://github.com/web-platform-tests/wpt.fyi/issues/1154

### CODE REVIEW

**Comment by lukebjerring** [comment]
@stephenmcgruer - the Travis CI build will automatically deploy a staging enviroment including the fix, which you can use to verify it's WAI.

**Comment by stephenmcgruer** [comment]
Wow, thanks for the quick PR!

This appears to work for the 'overall' results view (e.g. https://sticky-dot-wptdashboard-staging.appspot.com/results/?label=master&product=chrome%5Bexperimental%5D&product=edge&product=firefox%5Bexperimental%5D&product=safari%5Bexperimental%5D&aligned) but not for results for a specific test (e.g. https://sticky-dot-wptdashboard-staging.appspot.com/results/fetch/data-urls/processing.any.worker.html?label=master&product=chrome%5Bexperimental%5D&product=edge&product=firefox%5Bexperimental%5D&product=safari%5Bexperimental%5D&aligned). Maybe there's somewhere else the style needs applied?

File: webapp/components/abstract-test-file-results-table.js [hunk 1]
```diff
@@ -6,70 +6,73 @@
...
   }
 
+  static get template() {
+    return html`
+<style include="wpt-colors">
+  table {
+    width: 100%;
+    border-collapse: collapse;
+  }
+  th {
```
Comment (lukebjerring) [inline webapp/components/abstract-test-file-results-table.js]: This `th` style is the only actual change; the remainder is just cleanup of the auto-upgrade tool for Polymer 2 => 3 from #982 a while back.

File: webapp/components/abstract-test-file-results-table.js [hunk 2]
```diff
@@ -6,70 +6,73 @@
 import { TestRunsBase } from './test-runs.js';
 import { WPTColors } from './wpt-colors.js';
 import './test-file-results.js';
-const $_documentContainer = document.createElement('template');
+import { html } from '../node_modules/@polymer/polymer/lib/utils/html-tag.js';
```
Comment (Hexcles) [inline webapp/components/abstract-test-file-results-table.js]:
```suggestion
import { html } from '@polymer/polymer/polymer-element.js';
```
Same problem as last time.

Comment (lukebjerring) [reply webapp/components/abstract-test-file-results-table.js]: So, this link is copied from another file. It's strange to me that it's broken in some files and that goes unnoticed - I'll send a separate PR for changing all of them.

Comment (Hexcles) [reply webapp/components/abstract-test-file-results-table.js]: Yeah that's indeed strange. Looks like it's only broken when you directly import `html-tag.js` in *tests*. Might have something to do with the dynamic import path overwrite of `wct-server`.

Comment (lukebjerring) [reply webapp/components/abstract-test-file-results-table.js]: https://github.com/web-platform-tests/wpt.fyi/pull/1157

**Comment by stephenmcgruer** [comment]
The latest version deployed to the staging server does appear to have sticky headers in both places, thanks!

**Pull request merged**